### DISCUSSION TITLE
Discontinuous phonon band at Γ point with LO-TO splitting

### DISCUSSION THREAD

**Discussion by zentenmiyata** [ttadano/alamode#145] [Q&A] [answered]
Dear Tadano-sensei,

Using OpenMX, I tried to calculate the Born effective charge from the Berry curvature to compute the LO-TO splitting(NONANALYTIC=2).
However, when the LO-TO splitting is considered, the phonon branch becomes discontinuous at the Γ point as shown in the figure.
Discontinuities in phonon bands have been discussed in past forums, "This is the expected behavior of the mixed space approach NONANALYTIC=2, which yields a discontinuous dispersion curve at q points that is not commensurate with the size of the supercell." tadano-sensei responded.
However, in this calculation, the phonon bands are calculated from a 2x2x2 primitive cell, and the Born effective charge is calculated from the 1x1x1 primitive cell, so the supercell size does not seem to matter.
Incidentally, when LO-TO splitting is not considered (NONANALYTIC=0), such discontinuous bands do not occur. The exchange correlation potential was used GGA-PBE + DFT-D3.
Thank you in advance for your help.

Best regards,
![pic1](https://github.com/ttadano/alamode/assets/117719045/8ec6f3be-f35b-432d-93e4-8ecaf3b6af62)

Miyata

**Comment by ttadano** [comment]
If this system is non-cubic, this result should be fine.

In non-cubic systems, dielectric properties can depend on the crystallographic axis. For example, the dielectric constant and polarization can be direction dependent. This results in anisotropy in the LO-TO splitting and yields a discontinuity at the Gamma point.

**Reply by zentenmiyata** [reply, marked as answer]
Dear Tadano-sensei,

Thank you very much for your prompt reply.
As you mentioned, this material is non-cubic and highly anisotropic.　The anisotropy of this material in dielectric properties and polarization has been experimentally confirmed, so I think you are right.
Thank you very much.

Best regards,
Miyata
